### PR TITLE
add has activity planning option to Lessons

### PR DESCRIPTION
**Pull request by ericcf**
[#108391510]

### CODE REVIEW

File: app/views/lessons/_form.html.erb [hunk 1]
```diff
@@ -1,22 +1,41 @@
...
+    <%= f.label :day_in_treatment, class: "col-sm-2 control-label" %>
+    <div class="col-sm-10">
+      <%= f.number_field :day_in_treatment, class: "form-control", required: true %>
     </div>
+  </div>
+
+  <div class="form-group">
+    <%= f.label :has_activity_planning, class: "col-sm-2 control-label" %>
+    <div class="col-sm-10">
+      <div class="radio">
```
Comment (michaelwehrley) [inline app/views/lessons/_form.html.erb]: I haven't seen this before?  Why did you go with this approach?

**Comment by michaelwehrley** [comment]
:looking :good

File: app/views/lessons/_form.html.erb (hunk 1 above)
Comment (ericcf) [inline app/views/lessons/_form.html.erb]: Which approach are you referring to? I used their example markup:

http://getbootstrap.com/css/#checkboxes-and-radios

Comment (michaelwehrley) [inline app/views/lessons/_form.html.erb]: my memory fails me :-( thanks!

**Pull request merged**